README.md

```diff
@@ -212,6 +212,19 @@ You will likely see a number of create table statements (see data model section
 which tells you that if you apply these changes, that sql script will be applied to the sample db
 
 
+### Database servers with passwords
+
+  sem-apply --url postgresql://postgres@localhost/sample --pgpass
```

Comment:
also, I think we may want to consider renaming the parameter to "--password" - it actually has nothing to do with pgpass in psql - it is just the password.